libraries/iio/iio.c

```diff
@@ -83,17 +84,39 @@ struct element_info {
 struct iio_desc {
 	struct tinyiiod			*iiod;
 	struct tinyiiod_ops		*iiod_ops;
+	enum pysical_link_type	phy_type;
```

Comment:
inconsistent indentation

Reply:
done

Reply:
still looks inconsistent. maybe use just tabs or just spaces.

Reply:
![image](https://user-images.githubusercontent.com/16192750/87535012-7c8df800-c69f-11ea-8ddd-c542ef603631.png)

In the PR seems to be the old line.

Reply:
But if you look at the file changes it is ok